app/javascript/components/batch_update/components/PriceInput.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import TextInput from './TextInput'
+
+const PriceInput = ({ minPrice, maxPrice, onAddMinPrice, onAddMaxPrice }) => {
+  const onSetMin = value => {
+    let newValue = parseInt(value)
+    if (!isNaN(newValue)) {
+      onAddMinPrice(newValue)
+    }
+  }
+  const onSetMax = value => {
+    let newValue = parseInt(value)
+    if (!isNaN(newValue)) {
+      onAddMaxPrice(parseInt(value))
+    }
+  }
+
+  return (
+    <div>
+      <TextInput
+        placeholder={`Minimum Price${minPrice ? ': $' + minPrice : ''}`}
```

Comment:
We'll want to confirm, but I think(?) all prices are converted to USD for the ES index? (At least that's what I'm inferring from the various non-round-number figures that I _think_ I saw there). 

If that's right we'll want to include "USD" in the placeholder rather than "$"

Reply:
And as mentioned above, if the current app state includes a min price, we'll want to hide the min price input; ditto for max price.

(As for code and file naming & organization, I think we can follow the pattern for `SelectedCreatedAfterDate`
 / `SelectedCreatedBeforeDate`)

Reply:
Cool, I'll do this, thanks!

Reply:
So I've left it with `SelectedPrice` and I just pass the state key and the clearState methods separately for minPrice and maxPrice, rather than having two named components - is that cool, or am I missing something?